js/index.js

```diff
@@ -40,3 +40,120 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Navigation Bar
+
+let services = document.getElementsByTagName('a')[0];
+services.innerText = siteContent['nav']['nav-item-1']
+services.style.color = 'green';
+
+let product = document.getElementsByTagName('a')[1];
+product.innerText = siteContent['nav']['nav-item-2'];
+product.style.color = 'green';
+
+let vision = document.getElementsByTagName('a')[2];
+vision.innerText = siteContent['nav']['nav-item-3'];
+vision.style.color = 'green';
+
+let features = document.getElementsByTagName('a')[3];
+features.innerText = siteContent['nav']['nav-item-4'];
+features.style.color = 'green';
+
+let about = document.getElementsByTagNam('a')[4];
+about.innerText = siteContent['nav']['nav-item-5'];
+about.style.color = 'green';
+
+let contact = document.getElementsByTagName('a')[5];
+contact.innerText = siteContent['nav']['nav-item-6'];
+contact.style.color = 'green';
```

Comment:
This way works! I would stretch to see if you could use a loop of some sort to do the same thing.